.github/workflows/stale.yml

```diff
@@ -5,24 +5,23 @@ on:
     - cron: "30 13 * * *"
```

Comment:
what about `30 13 * * 1-5` so it only runs on weekdays?

Reply:
I like that